modules/retrieving-logs-from-a-rhacs-diagnostic-bundle.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_content-type: CONCEPT
+[id="retrieving-logs-from-a-rhacs-diagnostic-bundle_{context}"]
+= Retrieving logs from a RHACS diagnostic bundle
+
+You can also access Collector logs by downloading a diagnostic bundle from the {product-title} ({product-title-short}) user interface. Once you have downloaded the diagnostic bundle, you can inspect the logs for all the Collector pods. For more information, see xref:../configuration/generate-diagnostic-bundle.adoc[Generating a diagnostic bundle].
```

Comment:
**issue:** As a convention, we do not use xref's in modules for OpenShift docs. Which means we cant use the `xref` here. 

I suggest that we can move the content directly in the assembly and delete this file.

Reply:
Done